exporters/etw/include/opentelemetry/exporters/etw/.gitignore

```diff
@@ -0,0 +1 @@
+TraceLoggingDynamic.h
```

Comment:
Why need this?

Reply:
We can't share it in OpenSource yet. This is the place where I temporarily inject it. Once it's OSS, then we can remove it from `.gitignore` here.

Reply:
I'll resolve this conversation conversation. We will share this header in OSS ETA: April 2021 (almost there)>